packages/orchestrator-ui-components/src/components/WfoJsonCodeBlock/styles.ts

```diff
@@ -7,7 +7,20 @@ export const getStyles = (theme: EuiThemeComputed) => {
         borderRadius: theme.border.radius.medium,
     });
 
+    const euiBasicCodeBlockStyle = css({
```

Comment:
Is this basic in the sense the other one is extended or is it more like a simpleStyle?

Reply:
This refers to the simplified style (`isBasicStyle` prop is `true`)